sys/include/ztimer/config.h

```diff
@@ -114,6 +121,19 @@ extern "C" {
 #  endif
 #endif
 
+/**
+ * @brief   The minimum pm mode required for ZTIMER_LPTIMER to run
+ */
+#ifndef CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE
+#  if defined(CONFIG_ZTIMER_MSEC_REQUIRED_PM_MODE)
+#    define CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE CONFIG_ZTIMER_MSEC_REQUIRED_PM_MODE
+#  elif defined(CONFIG_ZTIMER_SEC_REQUIRED_PM_MODE)
+#    define CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE CONFIG_ZTIMER_SEC_REQUIRED_PM_MODE
+#  else
+#    define CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE ZTIMER_CLOCK_NO_REQUIRED_PM_MODE
+#  endif
+#endif
+
```

Comment:
this block should not be added (the other blocks a just there for backward compatibility) translating from x-SEC require PM_MODE-X is not correct since the PM is need by the timer that is doing the backend  -> if one writs a configuration for LPTIMER they may also set  CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE if they know.

translating from 
CONFIG_ZTIMER_MSEC_REQUIRED_PM_MODE
would never be reasonable since there is no old code that knew about LPTIMER before ->

do not add this block for LPTIMER
```suggestion
```

Reply:
maybe I should have wite in line 106 and 115 : this is only for backward compatibility and not supposed to be used by new code

Reply:
yes 
```suggestion
/**
 * @brief   The minimum pm mode required for ZTIMER_LPTIMER to run
 */
#ifndef CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE
#    define CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE ZTIMER_CLOCK_NO_REQUIRED_PM_MODE
#endif

```

Reply:
Done